dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -74,10 +74,18 @@ RUN apt-get -y purge   \
 
 # Clean up apt
 # Remove /var/lib/apt/lists/*, could be obsoleted for derived images
-RUN apt-get clean -y                     && \
-    apt-get autoclean -y                 && \
-    apt-get autoremove -y                && \
-    rm -rf /var/lib/apt/lists/* /tmp/*
+RUN apt-get clean -y         && \
```

Comment:
@YuriiHordynskyi you should submit it back to sonic as PR.

Reply:
ok. will create the PR if it's not present on the master branch.